packages/react/src/components/TextInput/next/TextInput.stories.js

```diff
@@ -81,3 +85,13 @@ export const WithLayer = () => {
 };
 
 export const Skeleton = () => <TextInputSkeleton />;
+
+export const Playground = (args) => (
+  <TextInput
+    {...args}
+    labelText="Text input label"
+    helperText="Optional help text"
+    id="text-input-1"
+    type="text"
+  />
```

Comment:
Added playground storybook to test

Reply:
Nice, thanks for adding that in @TannerS! We'll need to define the args somewhere in the storybook; take a look at `FluidTextInput` for an example:

```js
Playground.argTypes = {
  placeholder: {
    control: {
      type: 'text',
    },
    defaultValue: 'Placeholder text',
  }
  ...
  ...
  ...
};
```

There are some props that don't apply to `FluidTextInput` but do to `TextInput`, so it may not be a complete list. I believe you can copy most of these over to the `TextArea` story as well